rfcs/text/0000-definition-of-ecs-compliance.md

```diff
@@ -0,0 +1,282 @@
+# 0000: Definitions of ECS Compliance
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **1 (draft)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+<!--
+As you work on your RFC, use the "Stage N" comments to guide you in what you should focus on, for the stage you're targeting.
+Feel free to remove these comments as you go along.
+-->
+
+Events following [ECS guidelines and best practices](https://www.elastic.co/guide/en/ecs/current/ecs-guidelines.html) are described as "compliant". While the guidelines outlined in the ECS docs provide a general overview, this proposal aims to standardized what is and is not expected of an ECS-compliant event.
+
+This document's usage of the terms _must_, _must not_, _should_, _should not_, _required_, and _may_ are in accordance with [IETF RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## Fields
+
+The following sections briefly describe the required, suggested, and optional practices for complying with ECS. Later sections include more detailed examples of certain items.
+
+### Minimum requirements
+
+An ECS-compliant event MUST:
+
+* populate the `@timestamp` field with the date/time when the event originated.
+* set `ecs.version` to the ECS version this event conforms.
+* index all ECS fields using the data type defined in the schema. A different type from the same type family (e.g., substitute `text` for `match_only_text` or `keyword` for `wildcard`) may substitute.
+* use nested fields over dotted. ECS events should use nested objects, like `{ "log": { "level": "debug" }}`), over dotted field names, such as `{ "log.level": "debug" }`.
+
+### Recommended guidelines
+
+ECS-compliant events SHOULD:
+
+* map the contents of the original event to as many ECS fields as possible.
+* populate the top-level `message` field.
+* store the entire raw, original event in `event.original`. Indexing and doc_values are disabled on `event.original` to reduce store, but the value is retrieved from `_source`.
+* use an array if the field expects and array event if populating a single value.
```

Comment:
This line confuses me a little bit. I think you're just trying to say use an array if we specify array, but I wanted to double check

Reply:
Rephrased in https://github.com/elastic/ecs/pull/1868/commits/fa68b3b21ced0b394a11f255b835ec6958e3e791. Is the rephrasing an improvement?

Reply:
Yes! Thanks for the update